### DISCUSSION TITLE
Discrepancy results in velocity and expression plots using dynamical model

### DISCUSSION THREAD

**Discussion by qoiopipq** [theislab/scvelo#472] [Q&A]
Great package to use! 

I tried both stochastic and dynamical models for my data. But I observed discrepancy in velocity and expression plots under dynamical model. My understanding is it doesn't matter a gene shows positive or negative velocity, the gene must be expressed first.
 
Clustering plot:
![image](https://user-images.githubusercontent.com/16911467/100697705-d1ca6c00-33ea-11eb-9183-d660cc2cd07e.png)

Under dynamical model, these two genes showing as up-regulated in almost all clusters in velocity plots. But in the expression plots, the first gene was only expressed in the bottom clusters while the second gene was only expressed in the top clusters.

Dynamical model:
![image](https://user-images.githubusercontent.com/16911467/100696304-5915e080-33e7-11eb-8af5-e8b55c144cf3.png)

Similar results were observed in the top10 likelihood genes (these two genes were among top10 genes). 
When I checked these two genes under stochastic model, velocity and expression plots correlated with each other pretty well. 

Stochastic model:
![image](https://user-images.githubusercontent.com/16911467/100697389-fa059b00-33e9-11eb-8fdc-5e1a00183df8.png)

I don't understand why in dynamical model, those genes showing positive velocity while they were not expressed?

**Comment by WeilerP** [comment]
@qoiopipq, based on the phase portraits you showed, I believe you should not be too confident in the results of your velocity analysis:

1. The phase portraits are a bit noisy
2. Their is no clear characteristic almond/football shape present in the data you showed. Instead, the phase portraits look like they can be divided into two parts. For `Zeb2` the navy blue and pink cluster influence the algorithm substantially; in the case of `Relb` it is the light blue cells. In addition, the second phase portrait is more linear than not, i.e. their is substantial ambiguity.

It looks to me that some observations in which you consider gene `Zeb2`/`Relb` to be not expressed are actually expressed. Did you plot these observations (in your case the yellow ones) in the phase portrait to check that their expression really is 0? Also, is this the expression in unspliced, spliced or both combined?

**Comment by denvercal1234GitHub** [comment]
@qoiopipq -- Did you figure out the expression that could underlie the discrepancy? Or did you decide your data was not compatible for RNA Velocity?
